docs/sources/zoom/README.md

```diff
@@ -82,6 +82,35 @@ Once the scopes are added, click on `Done` and then `Continue`.
 
 6. Activate the app
 
+## Zoom AI Metric Snapshot Bulk
+
+Psoxy can pseudonymize Zoom AI Metrics Snapshot CSV data.
+
+The default proxy rules for `zoom-ai-snapshot-bulk` will pseudonymize `User Name` and `Email`, redacting `Department`
+
+```hcl
+custom_bulk_connector_rules = {
+    "zoom-ai-snapshot-bulk" = {
+        source_kind               = "zoom-ai-snapshot",
+        worklytics_connector_id   = "bulk-import-psoxy"
```

Comment:
i'd prefer this as `zoom-ai-metrics` bc:
  - it's really just metrics; not a "snapshot" of anything (where repeated calls of the same endpoint will return different values)
  - per comments on the tickets, I prefer connectors in worklytics be organized by source, then connection method - rather than the reverse